cli-scripts/get_kura.rb

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env ruby
+
+# SakeTimesの酒蔵一覧を使ってNDJSONを作るスクリプト
+#
+# NDJSONには1行に1データが入っており改行で区切られている。
+# 内容は
+#  {name: String(酒蔵名）, region: String(都道府県名)}
+# となっている。
+#
+# このデータを利用するには以下のように、1行ずつJSON.parseをすればよい。
+#   File.foreach(filename, chomp: true) do |line|
+#     json =  JSON.parse(line)
+#     p json
+#   end
+#
+# SakeTimesのウェブページは各都道府県（海外含む）ごとに別ページになっている。
+# そこで、まずは都道府県の一覧を取得してから、実際の酒蔵の名前を取得する
+#
+#   https://jp.sake-times.com/sakagura" 都道府県一覧のページ
+#      |
+#      +-- /hokkaido 北海道の酒蔵一覧
+#      +-- /aomori   青森県の酒蔵一覧
+#      ...
+
+require "open-uri"
+require "nokogiri"
+require "json"
+
+# aタグオブジェクトから地域とURLのハッシュを作る
+#
+# @param a_tag [Nokogiri::XML::Element] Nokogiriで生成したHTMLのaタグオブジェクト
+# @return [Array<Hash<Symbol => String>>] 地域とURLのハッシュ
+def a_to_region(a_tag)
+  url = a_tag.attributes["href"].value
+  # "北海道（13）"のようになっているので、括弧と数字を削る
+  region = a_tag.content.sub(/\uff08.*\uff09/, "")
+  { region: region, url: url }
+end
+
+# SakeTimesの酒蔵一覧ページから、地域とURLのハッシュを作る
+#
+# 地域は都道府県と海外広域（ex. アジア）がある。
+#
+# @example 実行例
+#   request_regions
+#     #=> [{ region: "北海道",
+#            url: "https://jp.sake-times.com/sakagura/./hokkaido"] },
+#          ...]
+#
+# @return [Array<Hash<Symbol => String>>] 地域とURLのハッシュの配列
+def request_regions
+  base_url = "https://jp.sake-times.com/sakagura/"
+  html = URI.parse(base_url).open
+  doc = Nokogiri::HTML(html).css("ul.sakayagura-list li a")
+  doc.map { |a| a_to_region(a) }
```

Comment:
a_to_regionメソッド、こんくらいなら分けなくてもいいんじゃない？
```suggestion
  doc.map { |a| 
    {
      url: a_tag.attributes["href"].value,  # "北海道（13）"のようになっているので、括弧と数字を削る
      region: a_tag.content.sub(/\uff08.*\uff09/, "")
    }
   }
```

Reply:
`map`内に書くと以外にわかりにくかった。
なるべく名前つけて別にするほうがよかった。

Reply:
`tr_to_HOGE`にしているし、ここも別名つけといたほうが統一がとれてよい

Reply:
ok